models/demos/tg/llama3_70b/tt/llama_decoder_galaxy.py

```diff
@@ -234,10 +234,9 @@ def decode_forward(
         start_pos: int,
         attn_masks: List[ttnn.Tensor],
     ) -> List[ttnn.Tensor]:
-        xs_interleaved = ttnn.to_memory_config(xs, memory_config=ttnn.DRAM_MEMORY_CONFIG)
-
-        attn_norm_out = self.tt_distributed_rmsnorm(
-            xs_interleaved,
+        attn_norm_out = tt_sharded_distributed_rmsnorm(
```

Comment:
We could either make this a class method or - probably better move self.tt_distributed_rmsnorm to llama common, then self.tt_distributed_rmsnorm and tt_sharded_distributed_rmsnorm have the same signature.